server/lib/pbench/__init__.py

```diff
@@ -251,6 +251,11 @@ def __init__(self, cfg_name):
                 except (NoOptionError) as exc:
                     raise BadConfig(str(exc))
 
+        try:
+            self.logging_level = self.conf.get("logging", "logging_level")
```

Comment:
Can we call it `default_logging_level` instead? After all, individidual applications can set their logging level differently.

Reply:
sure